### PR TITLE
Improvements in payment for app movil

### PR DESCRIPTION
**Pull request by vutreras**

### CODE REVIEW

File: onepay/controllers/front/commit.php [hunk 1]
```diff
@@ -15,16 +15,19 @@ public function postProcess() {
...
+            OnepayBase::setSharedSecret($sharedSecret);
             OnepayBase::setCurrentIntegrationType($endpoint);
 
             $externalUniqueNumber = Tools::getValue('externalUniqueNumber');
             $occ = Tools::getValue('occ');
 
             try {
-                $options = new Options();
+
+                $options = new Options($apiKey, $sharedSecret);
```
Comment (goncafa) [inline onepay/controllers/front/commit.php]: Why you need to set the apiKey and sharedSecret here too if you have just seted those before?

File: onepay/controllers/front/transaction.php [hunk 2]
```diff
@@ -67,13 +69,14 @@ public function initContent() {
             }
 
             try {
-                $options = new Options();
+
+                $options = new Options($apiKey, $sharedSecret);
```
Comment (goncafa) [inline onepay/controllers/front/transaction.php]: same question here.

**Review by goncafa — CHANGES_REQUESTED** [review]

File: onepay/controllers/front/commit.php (hunk 1 above)
Comment (vutreras) [reply onepay/controllers/front/commit.php]: is to omolog all the plugins, they are all like this

File: onepay/controllers/front/transaction.php (hunk 2 above)
Comment (vutreras) [reply onepay/controllers/front/transaction.php]: is to omolog all the plugins, they are all like this

File: onepay/controllers/front/commit.php (hunk 1 above)
Comment (vutreras) [reply onepay/controllers/front/commit.php]: Changed

File: onepay/controllers/front/transaction.php (hunk 2 above)
Comment (vutreras) [reply onepay/controllers/front/transaction.php]: Changed

**Review by goncafa — APPROVED** [review]

**Pull request merged**